UIKit-HW/UIKit-HW/FoodComponentsViewController/ViewController/FoodComponentsViewController.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  FoodComponentsViewController.swift
+//  practice(Pizza components)
+//
+//  Created by Матвей Белков on 29.06.2021.
+//
+
+import UIKit
+
+//MARK: - FoodComponentsVCDismissProtocol
+protocol FoodComponentsVCDismissProtocol {
+    func dismissVC()
+}
+
+final class FoodComponentsViewController: UIViewController {
+    
+    // MARK: - Private Property
+    @IBOutlet private var nameFoodLabel: UILabel!
+    @IBOutlet private var foodImageView: UIImageView!
+    @IBOutlet private var componentsFoodLabels: [UILabel]!
+    @IBOutlet private var componentSwitchs: [UISwitch]!
+    @IBOutlet private var enterButton: UIButton!
+    
+    // MARK: - Public Property
+    var food: Food?
+    var delegate: FoodVCPopToRootProtocol?
```

Comment:
```suggestion
   weak var delegate: FoodVCPopToRootProtocol?
```
грубая ошибка, которая повлечет за собой утечку памяти